Update bulkhead-content.md (#4493)
MicrosoftDocs/architecture-center: docs/patterns/bulkhead-content.md

## Issues and considerations

> - Define partitions around the business and technical requirements of the application. Partition boundaries should ideally align with the bounded context. Check [using tactical DDD to design microservices](https://learn.microsoft.com/azure/architecture/microservices/model/tactical-ddd) for further information.

Reviewer: ```suggestion
- Define partitions around the business and technical requirements of the application.
- If using [tactical DDD to design microservices](/azure/architecture/microservices/model/tactical-ddd), partition boundaries should align with the bounded contexts.
```
